src/main/java/forestry/core/config/Config.java

```diff
@@ -31,6 +31,7 @@
 
 public class Config {
 
+    public static Boolean DISABLE_BUTTERFLY;
```

Comment:
Please follow the style of the rest of the code - camelCase config variable name, use a `boolean` not a boxed `Boolean`, put this field below the categories together with other configs.